### PR TITLE
Program to convert a given string to Pig Latin

### PR DESCRIPTION
**Pull request by RohanSardar**
This is a program to convert a user given string to its respective Pig Latin form

As per wikipedia (link: https://en.wikipedia.org/wiki/Pig_Latin#Rules) For words that begin with consonant sounds, all letters before the initial vowel are placed at the end of the word sequence. Then, "ay" is added, as in the following examples: 
"pig" = "igpay"
"latin" = "atinlay"
"banana" = "ananabay"

When words begin with consonant clusters (multiple consonants that form one sound), the whole sound is added to the end when speaking or writing. 
"friends" = "iendsfray"
"smile" = "ilesmay"
"string" = "ingstray"

For words that begin with vowel sounds, one just adds "hay", "way" or "yay" to the end. Examples are: 
"eat" = "eatway"
"omelet" = "omeletway"
"are" = "areway"

### Describe your change:

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [x] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

File: strings/pig_latin.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+def pig_latin(sentence):
```
[the remark this answers is not in the document]
Comment (RohanSardar) [reply strings/pig_latin.py:1]: def function() -> str:

**Comment by cclauss** [comment]
This has been tried a few times https://github.com/search?q=repo%3ATheAlgorithms%2FPython+pig&type=pullrequests

This will need type hints on the function call and doctests and pass the three checks below.

File: strings/pig_latin.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
```
[the remark this answers is not in the document]
Comment (RohanSardar) [reply strings/pig_latin.py]: def pig_latin(sentence:str) -> str:

[the remark this answers is not in the document]
Comment (RohanSardar) [reply strings/pig_latin.py]: # pig_latin.py

def pig_latin(sentence:str) -> str:

    """Compute the piglatin of a given string.

    Usage examples:
    >>> pig_latin("pig")
    'igpay'
    >>> pig_latin("latin")
    'atinlay'
    >>> pig_latin("banana")
    'ananabay'
    >>> pig_latin("friends")
    'iendsfray'
    >>> pig_latin("smile")
    'ilesmay'
    >>> pig_latin("string")
    'ingstray'
    >>> pig_latin("eat")
    'eatway'
    >>> pig_latin("omelet")
    'omeletway'
    >>> pig_latin("are")
    'areway'

    """

    sentence = sentence.lower()
    length = len(sentence)
    if sentence[0] in "aeiou":
        result = sentence + "way"
    else:
        for i in range(length):
            if sentence[i] in "aeiou":
                break
        result = sentence[i:] + sentence[:i] + "ay"
    return result

statement = input("Enter a string: ")
answer = pig_latin(statement)

print(f"The PIG LATIN of the entered string is \"{answer}\"")

**Review by cclauss — APPROVED** [review]
Nice!  See the changes that I made.

**Pull request merged**